src/test/scala/cromwell/MainSpec.scala

```diff
@@ -9,17 +9,16 @@ import akka.util.Timeout
 import better.files._
 import cromwell.util.FileUtil._
 import cromwell.util.SampleWdl
-import cromwell.util.SampleWdl._
+import cromwell.util.SampleWdl.{EmptyWorkflow, GoodbyeWorld, ThreeStep}
 import org.apache.commons.io.output.TeeOutputStream
 import org.scalatest.concurrent.TimeLimitedTests
 import org.scalatest.time.Span
-import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
+import org.scalatest.{Matchers, BeforeAndAfterAll, FlatSpec}
```

Comment:
I'll reorder :santa: 

I had to fix a lot of the imports manually (due to lost implicits and such), was probably done by hand